user_guide_src/source/general/errors.rst

```diff
@@ -148,6 +160,31 @@ Since v4.3.0, you can specify the HTTP status code for your Exception class to i
 
 When an exception implementing ``HTTPExceptionInterface`` is caught by CodeIgniter's exception handler, the Exception code will become the HTTP status code.
 
+.. _http-status-code-and-error-views:
+
+HTTP Status Code and Error Views
+================================
+
+The exception handler displays the error view corresponding to the HTTP status
+code, if it exists.
+
+For example, ``PageNotFoundException`` implements the ``HTTPExceptionInterface``,
+so its exception code ``404`` will be the HTTP status code. Therefore if it is
+thrown, the system will show the **error_404.php** in the **app/Views/errors/html**
+folder when it is a web request. If it is invoked via CLI, the system will show
+the **error_404.php** in the **app/Views/errors/cli** folder.
```

Comment:
I think this is the correct behavior, but the current implementation is not.
See  #8689